src/core/common/message.hpp

```diff
@@ -443,23 +443,25 @@ class Message : public Buffer
     int CopyTo(uint16_t aSourceOffset, uint16_t aDestinationOffset, uint16_t aLength, Message &aMessage) const;
 
     /**
-     * This method creates a copy of the current Message. It allocates the new one
-     * from the same Message Poll as the original Message and copies @p aLength octets of a payload.
+     * This method creates a copy of the current Message. It allocates the new one from the same message pool as the
+     * original Message and copies @p aLength octets of a payload.
      *
-     * The `Type`, `SubType`, `LinkSecurity` and `Priority` fields on the cloned message are also
-     * copied from the original one.
+     * The `Type`, `SubType`, `LinkSecurity` and `Priority` fields on the cloned message are also copied from the
+     * original one.
      *
      * @param[in] aLength  Number of payload bytes to copy.
      *
      * @returns A pointer to the message or NULL if insufficient message buffers are available.
+     *
      */
     Message *Clone(uint16_t aLength) const;
 
     /**
-     * This method creates a copy of the current Message. It allocates the new one
-     * from the same Message Poll as the original Message and copies a full payload.
+     * This method creates a copy of the current Message. It allocates the new one from the same message pool as the
```

Comment:
The "brief" should be on its own line.  More detailed description should be below.